data/sql/updates/pending_db_world/rev_1583667245426891200.sql

```diff
@@ -271,8 +270,8 @@ UPDATE `creature_template` SET `mindmg` = 354, `maxdmg` = 470, `DamageModifier`
 UPDATE `creature_template` SET `mindmg` = 305, `maxdmg` = 405, `DamageModifier` = 1 WHERE `entry` = 8127;
 -- Sandarr Dunereaver
 UPDATE `creature_template` SET `mindmg` = 189, `maxdmg` = 251, `DamageModifier` = 1 WHERE `entry` = 10080;
--- Dustwraith (Not listed in Beastiary) Src: https://wotlk.evowow.com/?npc=10081 / https://db.rising-gods.de/?npc=10081
-UPDATE `creature_template` SET `mindmg` = 583, `maxdmg` = 774, `DamageModifier` = 1 WHERE `entry` = 10081;
+-- Dustwraith (Not listed in Beastiary) Src: https://classicdb.ch/?npc=10081
+UPDATE `creature_template` SET `mindmg` = 64, `maxdmg` = 82 WHERE `entry` = 10081;
```

Comment:
something's wrong here, he's a lvl 48 boss, the original values looked high but here he looks like harmless and u forgot the damage modifier